ui-tests/src/test/java/io/jenkins/plugins/analysis/warnings/SmokeTests.java

```diff
@@ -163,6 +164,66 @@ private void verifyDetailsTab(final Build build) {
         assertThat(categoryPaginateButtons.size()).isEqualTo(1);
     }
 
+    private void verifyTrendCharts(final Build build) {
+        AnalysisResult analysisResult = new AnalysisResult(build, "checkstyle");
```

Comment:
Können Sie den Code nicht innerhalb von `verifyCheckStyle(build);` aufrufen? Dann wird das automatisch in Pipeline und Freestyle mitgetestet? Dadurch wird auch mehr klar, dass dieser Teil nur für CheckStyle geprüft wird.